pkg/cmd/config.go

```diff
@@ -874,6 +880,7 @@ addons:
 # apiEndpoint:
 #   host: '{{ .APIEndpointHost }}'
 #   port: {{ .APIEndpointPort }}
+#	alternativeNames: {{ .APIEndpointAlternativeNames }}
```

Comment:
Looks like indentation doesn't match host and port fields.

Reply:
Fixed